projects/plugins/boost/app/modules/optimizations/page-cache/Page_Cache.php

```diff
@@ -51,6 +59,34 @@ public function setup() {
 		add_action( 'update_option_jetpack_boost_status_' . Liar::get_slug(), array( $this, 'invalidate_cache' ) );
 	}
 
+	public function register_data_sync( Data_Sync $instance ) {
+		$page_cache_schema       = Schema::as_assoc_array(
+			array(
+				'bypass_patterns' => Schema::as_array( Schema::as_string() ),
+				'logging'         => Schema::as_boolean(),
+			)
+		);
+		$page_cache_error_schema = Schema::as_assoc_array(
+			array(
+				'code'      => Schema::as_string(),
+				'message'   => Schema::as_string(),
+				'dismissed' => Schema::as_boolean()->fallback( false ),
+			)
+		)->nullable();
```

Comment:
How do we decide when to move schemas to a separate class?

Reply:
Yeah, I think what you're saying makes sense. We can't have a strict rule about this. If it hinders readability - move it. Otherwise it's fine to be in the main feature class.